### DISCUSSION TITLE
few queries

### DISCUSSION THREAD

**Discussion by vasnt** [Roznoshchik/Lurnby#52] [Q&A] [answered]
- how this tool is different than glasp-social web highlighting?
- does lurnby has chrome/browser extension?
-

**Comment by Roznoshchik** [comment, marked as answer]
Hey. 

Hadn't heard of glasp before. It seems like a cool tool. 
Lurnby doesn't work like that at the moment, although there were plans for similar types of functionality, of reading wherever you want.

Lurnby lets you read everything in one place, and gives the most benefit in the organization and the review capabilities.

Although with the speed at which the Readwise team developed and is still developing, I would say that they should probably be the go to option. I just haven't had the time to work on Lurnby and it's fallen behind. 

There are chrome and firefox extensions, but they only make it easier to add things to your lurnby library, they don't expose any other functionality such as what is being shown in Glasp. Although they can, with a spare weekend or two.

https://chrome.google.com/webstore/detail/lurnby/dpobgbljepcemmnbfflkcplalobfllgn

https://addons.mozilla.org/en-US/firefox/addon/lurnby/
